src/main/java/app/model/agents/Cells/BooleanCell.java

```diff
@@ -0,0 +1,27 @@
+package app.model.agents.Cells;
+
+public class BooleanCell implements Cell
+{
+    private boolean explored;
+    private boolean obstacle;
+
+    public void explored()
+    {
+        explored = true;
+    }
+
+    public void obstacle()
+    {
+        obstacle = true;
+    }
+
+    public boolean getExploredState()
```

Comment:
consider changing to the java default getter method name for booleans:  `isExplored`